frontend/src/lib/email/EmailVerificationStatus.svelte

```diff
@@ -1,8 +1,9 @@
 <script lang="ts" context="module">
-  import { writable, type Writable } from 'svelte/store';
+  import type { Writable } from 'svelte/store';
+  import { defineContext } from '$lib/util/context';
 
-  export let requestedEmail: Writable<string | null> = writable();
-  export let emailResult: Writable<EmailResult | null> = writable();
```

Comment:
so this was a singleton on the server and that was causing issues?

Reply:
Right:
1) Person 1 verifies their email => `verifiedEmail = true` is persisted on the server
2) Person 2 loads a page
   a) SSR uses the persisted `verifiedEmail = true`
   b) CSR fixes it/resets it back to `verifiedEmail = false/null`